crates/wash-cli/src/common/registry_cmd.rs

```diff
@@ -167,12 +167,17 @@ pub async fn registry_push(
 
     spinner.finish_and_clear();
 
-    let mut map = HashMap::new();
-    map.insert("url".to_string(), json!(artifact_url));
-    Ok(CommandOutput::new(
-        format!("{SHOWER_EMOJI} Successfully validated and pushed to {artifact_url}"),
-        map,
-    ))
+    let mut map = HashMap::from_iter([
+        ("url".to_string(), json!(artifact_url)),
+        ("digest".to_string(), json!(digest)),
+    ]);
+    let text = if let Some(tag) = maybe_tag {
+        map.insert("tag".to_string(), json!(tag));
+        format!("{SHOWER_EMOJI} Successfully pushed {artifact_url} with {tag}: digest: {digest}")
+    } else {
+        format!("{SHOWER_EMOJI} Successfully pushed {artifact_url} with digest: {digest}")
```

Comment:
wdyt about using a newline here to separate the
```
{SHOWER_EMOJI} Successfully pushed {artifact_url}
```
line from the line with the digest? 

I could see this thing getting pretty long, and if all you're looking for is the digest, it would be easier to find when it was on a new line.

Reply:
Done!

Reply:
Rad, thanks!